### ISSUE TITLE
CI: torch_1.11 lane has not installed torch since the matrix rename in #4008

### ISSUE THREAD

**Issue by MohammedAlkindi** [huggingface/huggingface_hub#4775]
The `torch_1.11` CI lane has not installed torch since March, so the tests it exists to run have been skipping and the job has been green anyway.

`.github/workflows/python-tests.yml:95` still matches on `"Python 3.9, torch_1.11")`, but commit `4c7b420f` (#4008, 2026-03-31) renamed the matrix entry at line 50 to `"Python 3.10, torch_1.11"` and updated only the run-step arm at line 140. The install `case` has no `*)` default, so no arm matches and the torch install is skipped silently. The run step still selects `test_hub_mixin*` and `test_serialization.py`, those tests skip with "Test requires torch", and the job passes.

Your own run 31806513458 shows the difference between the two lanes:

```
build-ubuntu (3.12, torch_latest)             torch==2.13.0   46 passed / 47 passed
build-ubuntu (3.10, Python 3.10, torch_1.11)  no torch        25 passed + 21 skipped / 18 passed + 29 skipped
```

That is about 50 tests not running, and torch 1.11 back-compatibility unverified for 138 days. It matters for anyone pinned to `torch~=1.11` using `PyTorchModelHubMixin` or `save_torch_state_dict`, which is the case the lane was added for.

Locally the skip is easy to see:

```
cd src && python -m pytest ../tests/test_hub_mixin_pytorch.py -q -rs
21 skipped  --  Test requires torch
```

The fix is one token in the case label. I would rather flag it than open the PR blind, because I cannot run Python 3.10 here, and once torch 1.11 actually installs those 50 tests may not all pass. If they fail, that is 138 days of drift surfacing rather than anything the one-token change introduced, and you would probably want to decide how to handle it before a patch lands.

**Comment by cananoo** [comment]
I plan to prepare a focused PR for this issue: update the torch_1.11 install case arm to the current matrix label (Python 3.10, torch_1.11). I will keep the run-step selector unchanged, add a local static check that the matrix entry and install/run arms stay aligned, and run the relevant repository quality checks that are available here. This is limited to restoring the intended CI coverage.

**Comment by cananoo** [comment]
The automated check closed #4781 because this issue does not yet have a maintainer-confirmed approach. The proposed change is intentionally limited to aligning the install selector with the existing Python 3.10, torch_1.11 matrix label; the run-step selector already uses that label. @Wauplin @hanouticelina, could you confirm whether this one-line CI fix is the intended resolution? If so, I can reopen #4781 after confirmation.

**Comment by Wauplin** [comment]
thanks for the report and fix @MohammedAlkindi ! I've reopened and fixed https://github.com/huggingface/huggingface_hub/pull/4776

@cananoo no need to spam with PRs doing exactly what @MohammedAlkindi did in the first place. The filter is there to keep the repo more quiet.
